packages/renderer/src/lib/container/container-utils.ts

```diff
@@ -18,6 +18,8 @@
 
 import type { Port } from '@podman-desktop/api';
 import { ContainerIcon } from '@podman-desktop/ui-svelte/icons';
+// eslint-disable-next-line unicorn/prefer-node-protocol
+import { Buffer } from 'buffer';
```

Comment:
```suggestion
import { Buffer } from 'node:buffer';
```

We have a preference for `node:*` import, that's why they was a eslint warning

Reply:
I get an error and a white screen for PD if I use `node:buffer` instead of `buffer`
![Screenshot from 2024-07-25 08-47-35](https://github.com/user-attachments/assets/67bf816d-5f25-4826-b74e-65a9d1433d43)

Reply:
ah yes it's because you're on the frontend side

Reply:
you need to keep `buffer`

Reply:
Okey I see the issue. But why are we using Buffer to decode a base64 instead of `btoa` @benoitf  any idea ?

Reply:
I think it's because they're flagged as deprecated

Reply:
https://stackoverflow.com/a/70851350

As mention in the issue, only `btoa` on node has been deprecated not the one on DOM